content/blog/open-sourcing-workshops-on-demand-part-2-how-to-deploy-the-infrastructure.md

```diff
@@ -0,0 +1,499 @@
+
+T﻿he principle remains similar, with a few differences explained below.
+
+* After cloning the wod-install repository, y﻿ou will fork the following public private [repo](https://github.com/Workshops-on-Demand/wod-private.git) on Github under your own Github account (we will refer to it as `Account`).    
+* Next, clone the forked repo.    
+* Edit the `all.yml` and `<groupname>` files to customize your setup. T﻿his variable `<groupname>` defines possible backend server in your environement. By default, the project comes with a sample working file named `production` in `ansible/group-vars`. But you could have multiple. In our case, we have defined `sandbox`, `test`, `staging` and several `production` files, all defining a different backend environment. These files will be used to override the default values specified by the public version delivered as part of the default public installation.    
+* Commit and push changes to your repo.    
+* Create an `install.priv` file located in `install` directory when using a private repo (consider looking at [install.repo](https://github.com/Workshops-on-Demand/wod-backend/blob/main/install/install.repo) file for a better understanding of the variables).
+* Define the WODPRIVREPO and WODPRIVBRANCH variables as follows:    
+
+  * `WODPRIVBRANCH="main"`     
+  * `WODPRIVREPO="git@github.com:Account/Private-Repo.git wod-private"`    
+
+**Note:** When using a token
+
+Please refer to the following [url](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) to generate a `token` file in `install` directory of WoD-backend:
+
+* Edit the `install.priv` file located in `install` directory of wod-install:  
+
+  * Create line before variable declaration: ``token=`cat $EXEPATH/token` ``    
+  * Use the token in the url WODPRIVREPO="git clone https://user:$token@github.com/Account/wod-private.git wod-private"    
+
+Y﻿ou are now ready to perform the installation again to support a private repository. 
+
+Please note that this setup phase can be concurrent with the public setup phase. Indeed, the install script should detect the presence of the private repository owing to the presence of the install.priv file. It will automatically adjust the different scripts and variables to add the relevant content. It will actually overload some of the variables with private ones.
+
+Y﻿ou now have a working Workshops-on-Demand backend server in place. Congratulations! The next article in the series will help you better understand the lifecycle of the backend server. How does a workshop registration work from the backend server 's side? How do you manage this server on a daily basis? How and when do you need to update it ? All these questions will be answered in the next article. And from there, we will move to the frontend side of things and finally to a workshop's creation process.
```

Comment:
There's an extra space between the "'s" and "server" where it says "How does a workshop registration work from the backend server's side?"

For the last sentence, consider changing it to:

And from there, I will help you move to the frontend side of things..."